### ISSUE TITLE
Text elements should have a reasonable contrast ratio

### ISSUE THREAD

**Issue by azayneeva** [ameliejyc/ameliejyc.github.io#1]
See [AX_COLOR_01](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_color_01) for more information
